src/ila-mngr/u_rewrite_ila.cc

```diff
@@ -6,6 +6,10 @@
 #include <ilang/ila-mngr/u_abs_knob.h>
 #include <ilang/util/log.h>
 
+// enforce parent valid condition on child-decode
+#define VALID_STACK
+#undef VALID_STACK
```

Comment:
Default `OFF` as several application (like simulator generation) do not comply to this.